extensions/amp-analytics/amp-analytics.md

```diff
@@ -476,6 +476,13 @@ In addition, all of the variables supported by
 [AMP HTML Substitutions Guide](../../spec/amp-var-substitutions.md)
 are also supported.
 
+Variables are resolved asynchronously and can delay the request until they are
+fulfilled. For example, some metrics such as Cumulative Layout Shift and
+Largest Contentful Paint are calculated after the page is hidden. For First
+Input Delay, it is resolved after the user interacts with the page. For this
+reason these metrics might not be suitable for use with all triggers (for
+example, on timer or visible).
```

Comment:
Looks good: QQ though, did we add a specific trigger for when all these values have been captured?

Reply:
No specific trigger, however using something like page view would allow the trigger to be fired immediately, and only wait for the CWV macros to resolve, before sending.